test/blogtoblog/BlogToBlogImport.spec.ts

```diff
@@ -69,3 +69,52 @@ describe('BlogToBlogImporter#buildRecommendedArticlesTable tests', () => {
       `<main>${div}${div}${div}${div}${div}</main>`);
   });
 });
+
+describe('BlogToBlogImporter#buildMetadataTable tests', () => {
+  const test = (input: string, expected: string) => {
+    const { document } = (new JSDOM(input)).window;
+    getImporter().buildMetadataTable(document, document, document);
+    strictEqual(document.head.innerHTML + document.body.innerHTML, expected);
```

Comment:
after much trial and error, this is what I can up that would actually allow me to run the tests. let me know what you think.